.rspec

```diff
@@ -1 +1,3 @@
---colour
+--color
+-r spec_helper
+-f documentation
```

Comment:
missing empty line

Reply:
fixed in https://github.com/spree-contrib/spree_slider/pull/50/files